pkg/config/config.go

```diff
@@ -204,6 +204,7 @@ type Endpoint struct {
 // Warnings represent the warnings in the config
 type Warnings struct {
```

Comment:
Maybe for another PR: should we rename this `Warning` instead of `Warnings` since this struct only contains a single message ? Do we plan on having more than 1 warning in the future?

Reply:
I think we need to revisite this struct entirely but in another PR.